doc/guidelines/code_style.md

```diff
@@ -1,8 +1,19 @@
 # Code Style
 
+## Formatting
 
-- do not use prettier for formatting
-- use the [barrel pattern](./barrel-pattern.md)
-- files in a module should not import from their own index file (except tests, demo pages)
-- tests and demo pages (+page.svelte) should import from the module's index to ensure that the module's public API is used
-- files outside of a module should always import from the module's index
\ No newline at end of file
+Do not use prettier for formatting, we use basic styling for now (which are defined in the [project settings](/.vscode/settings.json) for VSCode):
+* tabs is used for indentation
+* tabulation size = two spaces
```

Comment:
If we use tabs, does it matter how big they are?
The whole point of tabs is that everybody can set them as they like :)

Reply:
You are right, and this was written prior the introduction of BiomeJS, I will update this part too.